build/v1/types.go

```diff
@@ -661,6 +661,21 @@ const (
 	// warn to the build output instead of failing when unsupported syntax is detected. This
 	// policy is experimental.
 	ImageOptimizationSkipLayersAndWarn ImageOptimizationPolicy = "SkipLayersAndWarn"
+
+	// ImageOptimizationDaemonless is an experimental policy which builds an image without
+	// using a docker engine, typically by running a build tool directly inside of the
+	// builder container, using a single layer for all content changes relative to the base
+	// image.
+	ImageOptimizationDaemonless ImageOptimizationPolicy = "Daemonless"
+
+	// ImageOptimizationDaemonlessWithLayers is the same as ImageOptimizationDaemonless, but
+	// producing a separate layer for each instruction in the Dockerfile.
+	ImageOptimizationDaemonlessWithLayers = "DaemonlessWithLayers"
```

Comment:
missing "imageoptimizationpolicy" type.  same for the one below.

Reply:
Ah, thanks.  Will fix that.